documentation/src/content/docs/en/essentials/work-with-async.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Asynchronous Operations in effector
+description: How effects help work with asynchronous operations in effector
+---
+
+# Asynchronous Operations in effector using Effects
```

Comment:
Also, I need to make attention on header-ids.
Each header should have an unique and stable ID:

```md
## Methods (#methods)
```

Otherwise, when headers is reordered, ID can be changed from (#methods-1) to (#methods-2) automatically.

Also, sometimes, headers fixed or renamed, but their ID should not change at all. It must be as stable as we can do it so.

For API section ID-schema used: `#section-methodName-overrideParams`.

For other articles you can choose any type of convention to name each header-ID. Requirements:
1. Each header ID on the page should be unique
2. Header ID on each translated page should be the same (#async-effects the same on en/ru/fr/etc)
3. Header IDs should be consistent for the whole documentation section

Reply:
If no header-ID is set, there is no way to make link to it. That's how it's meant to be. Each header ID should be manually designed and set, without accidental breaks for the reasons above.

Reply:
got it, but what about the first heading, should we add a link to it?
it looks to me like the heading of a page that doesn’t need a link.

Reply:
Why not to be consistent?

Reply:
> Why not to be consistent?

in that sense I agree, it just seems redundant